content/en/cloudcraft/getting-started/datadog-integration.md

```diff
@@ -0,0 +1,51 @@
+
+## Datadog Single Sign-On (SSO)
+
+Cloudcraft allows users to sign up and log in using their Datadog account. This integration offers a unified experience, linking your Datadog monitoring data seamlessly with your Cloudcraft architecture diagrams.
+
+### Signing up with Datadog SSO
+
+To get started, choose the **Sign up with Datadog** option during the Cloudcraft sign-up process. After signing up, you can log in to Cloudcraft using your Datadog credentials. This not only simplifies the login process but also enables integration between the two platforms.
+
+By using Datadog SSO, you automatically gain access to:
+
+- **Cross-platform functionality**: Move effortlessly between Cloudcraft and Datadog to analyze your cloud infrastructure and its performance.
+- **Automated cloud account synchronization**: Cloud accounts configured in Datadog sync automatically with Cloudcraft, giving you a complete view of your infrastructure in both platforms.
+
+### Enabling Datadog SSO for existing accounts
+
+If you originally signed up with a different login method, for example, Google SSO or a standard username and password, you won’t have access to the full set of Datadog integration features. To switch to Datadog SSO, [contact the Cloudcraft support team][1] and they will assist you in converting your account.
+
+## Sync cloud accounts
+
+<div class="alert alert-info">This feature supports only Amazon Web Services (AWS) accounts. Syncing with Azure or other cloud providers is not available at this time.</div>
+
+The integration between Cloudcraft and Datadog streamlines cloud account management, allowing your cloud accounts to sync automatically between the two platforms. After adding your cloud account to Datadog, it is automatically synced with Cloudcraft, so there's no need to add the account separately in Cloudcraft.
+
+### How it works
+
+1. **Add your cloud account to Datadog**: Once you integrate your AWS account with Datadog, Cloudcraft will automatically recognize the account without any further action required on your part.
+2. **Resource Collection**: Ensure that **Resource Collection** is enabled in Datadog for this feature to function properly. Without this setting, the sync between Datadog and Cloudcraft will not work.
+
+For detailed instructions on how to add your AWS account to Datadog and enable Resource collection, please refer to the [AWS integration documentation][2].
+
+## The Bits menu
+
+The Bits menu in Cloudcraft is your gateway to accessing relevant Datadog insights from any resource in your architecture diagram. Whether you need to look at logs, view APM traces, or analyze metrics, the Bits menu offers seamless, context-driven navigation from Cloudcraft to Datadog with just a click.
+
+For more detailed information on how to use the Bits menu, please refer to our full documentation:
+
+- [Using the Bits menu][3]
```

Comment:
I would remove this line, integrating the link into the previous sentence as I suggested in the line above -- just more typical of our style.